src/SOS/SOS.UnitTests/DumpGenerationFixture.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.Diagnostics.TestHelpers;
+using Microsoft.Win32;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using Xunit;
+
+public class DumpGenerationFixture : IDisposable
+{
+    private static readonly string _root = RuntimeInformation.ProcessArchitecture == Architecture.X86 ? @"SOFTWARE\WOW6432Node\" : @"SOFTWARE\";
```

Comment:
There's one missing for arm64

Reply:
I assume that arm64 is the same as x64 "SOFTWARE". Is that true?